Source/Particles/LaserParticleContainer.cpp

```diff
@@ -286,6 +289,8 @@ LaserParticleContainer::InitData ()
 {
     if (!m_enabled) return;
 
+    m_laser_injection_box= Geom(0).ProbDomain();
```

Comment:
I'm not sure that this is still needed, since we now call `Restart` instead of `InitData` when restarting.

Reply:
Yes, indeed. 
However, where is the `Restart` function implemented? I am correct that this is part of `amrex` (not WarpX)?
Also: does `Restart` always call `PostRestart` under the hood?

Reply:
In that case, I think that we would have to call `m_laser_injection_box= Geom(0).ProbDomain();` inside `PostRestart`.

Reply:
`ParticleContainer::Restart` is implemented in amrex. `PostRestart` is all implemented in WarpX, and it is called by WarpX in `WarpXInitData.cpp`, after all checkpoints are read.